src/webots/gui/WbMultimediaStreamingLimiter.cpp

```diff
@@ -44,8 +44,6 @@ QSize WbMultimediaStreamingLimiter::fullResolution() const {
 void WbMultimediaStreamingLimiter::recomputeStreamingLimits(int skippedImages) {
   mIsStopped = false;
   mResolutionChanged = false;
-  if (mIsStopped)
```

Comment:
This should be the only change where the behavior is different than before.
I tested on my system and it still works as expected, but locally the connection is good.